config_!linux.go

```diff
@@ -0,0 +1,5 @@
+//go:build !linux
```

Comment:
Can we use a different filename?

Reply:
sure thing; the "exclamation mark" seems stupid now that i am seeing it the second time. :D

Reply:
I had a look into this. 

Looking at the GO reference : https://pkg.go.dev/cmd/go#hdr-Build_constraints  
It seems that the `xxx_!linux.go` notation for files goes hand in hand with the `//go:build !linux` compile header.

There shouldn't be any difference between build contraints in file names and compile header.

It still is possible to rename the file with the drawback to handle all the different OSes in runtime.
And being unable to provide any os-spedific functions like `SystemConfigFileFinder` (in case of linux)

Reply:
I updated to filename to one without the special char; the `//go:build !linux` compile header takes care of the build constraint.